src/app/profile/payment-methods/payment-methods.tpl.html

```diff
@@ -0,0 +1,34 @@
+<loading-overlay ng-if="$ctrl.loading"></loading-overlay>
```

Comment:
This seems to take over the entire page, including the menu so a user can't navigate away. Can you nest this farther in so it only covers the panel? Or it might be nice to keep the Add New Payment Method button accessible during loading... idk.